src/wakelock.js

```diff
@@ -22,22 +22,26 @@ var Util = require('./util.js');
  */
 function AndroidWakeLock() {
   var video = document.createElement('video');
+  video.setAttribute("loop", "");
 
-  video.addEventListener('ended', function() {
-    video.play();
-  });
+  function addSourceToVideo(element, type, dataURI) {
```

Comment:
There are no spaces before function parentheses anywhere else in the file so I'm not sure this would be consistent?